### PR TITLE
Create home screens and mood query component 

### LINKED ISSUE #5: Profile button component
## User story
As a **user**,
I want to **view and edit my profile.**  

## Notes/Links
Bildschirmfoto 2020-11-24 um 11 52 27

## Acceptance criteria
- [ ] Create link
- [x] Implement icon

### LINKED ISSUE #7: Interaction component
## User story
As a **developer**,
I want to implement **components which interact with the user.**  
This is helpful to call up the health status and give advantages advices. 

## Notes/Links
Bildschirmfoto 2020-11-24 um 15 57 16
Bildschirmfoto 2020-11-24 um 15 57 30
Bildschirmfoto 2020-11-24 um 15 57 40

## Acceptance criteria
- [x] Create health component 
- [ ] Create health update message

### LINKED ISSUE #8: Home screen
## User story
As a **developer**,
I want the users **to start always on home whenever they start the app.**  
This screen will help to interact/communicate with every user and share or collect information throughout the fasting process. 

## Notes/Links
Bildschirmfoto 2020-11-24 um 10 35 46

## Acceptance criteria
- [x] Implement profile button component
- [x] Implement interaction component
- [x] Implement navbar component

### PR DESCRIPTION
**Pull request by ReneSchmitz**
### Heroku
https://fabstr-pr-35.herokuapp.com/
### Useful Link/s
https://reactjs.org/docs/conditional-rendering.html
### Closed Issue/s
close #5 
close #7 
close #8

### CODE REVIEW

File: client/src/components/Mood.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import styled from "styled-components/macro";
+import smile from "../assets/icon/smile.svg";
+import sad from "../assets/icon/sad.svg";
+import { Link } from "react-router-dom";
+
+const Header = styled.h1`
```
Comment (phlgr) [inline client/src/components/Mood.js:6]: Same as the above

File: client/src/components/Sadness.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components/macro";
+import smile from "../assets/icon/smile.svg";
+import sad from "../assets/icon/coloredSad.svg";
+import { Link } from "react-router-dom";
+
+const Header = styled.h1`
```
Comment (phlgr) [inline client/src/components/Sadness.js:6]: Same as the above

File: client/src/components/Sadness.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  return (
+    <>
+      <Header>Das ist ok!</Header>
+      <Text>Gönn’ dir heute viel Ruhe.</Text>
+      <Container>
+        <Link to="/">
+          <Happiness src={smile} alt="happy" />
+          <Sadness src={sad} alt="sad" />
+        </Link>
+      </Container>
```
Comment (phlgr) [inline client/src/components/Sadness.js:45]: This and the Happiness component only seem to differentiate in the text. You could build a single component and pass the text via props

**Review by phlgr — CHANGES_REQUESTED** [review]

File: client/src/components/Sadness.js (hunk 3 above)
Comment (ReneSchmitz) [reply client/src/components/Sadness.js:45]: @phlgr Thanks for your review. But I'm going back and forth with it. I can not find a solution to change the text elements by clicking the "Happiness" or the "Sadness" button. Do you have more information to add?

File: client/src/components/Happiness.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components/macro";
+import smile from "../assets/icon/coloredSmile.svg";
+import sad from "../assets/icon/sad.svg";
+import { Link } from "react-router-dom";
+
+const Header = styled.h1`
```
[the remark this answers is not in the document]
Comment (ReneSchmitz) [reply client/src/components/Happiness.js:6]: Thanks!

File: client/src/components/Mood.js (hunk 1 above)
Comment (ReneSchmitz) [reply client/src/components/Mood.js:6]: Noted with thanks! 🙏

**Pull request merged**